modelopt/torch/quantization/plugins/transformer_engine.py

```diff
@@ -58,3 +59,58 @@ def te_quantized_linear_fn(package, func_name, self, *args, **kwargs):
 
     # Override the quantized linear function
     _quantized_linear_fn = te_quantized_linear_fn
+
+
+# Register the public te.pytorch.GroupedLinear class
+@QuantModuleRegistry.register({te_grouped_linear.GroupedLinear: "te_GroupedLinear"})
+class _QuantTEGroupedLinear(_ParallelLinear):
+    _functionals_to_replace = [
+        (te_grouped_linear._GroupedLinear, "forward"),
+        (te_grouped_linear._GroupedLinear, "apply"),
+    ]
+
+    def _setup(self):
+        # GroupedMLP stores the weights as weight0, weight1, etc. To run setup in order to
+        # initialize the quantizer states, self.weight is used to extract shape, dtype etc. Assigning
+        # self.weight0 to self.weight to run the quantizer states initialization.
+        self.weight = self.weight0
```

Comment:
nit: may be assert that self.weight does not exist?
```suggestion
        assert not hasattr(self, "weight")
        self.weight = self.weight0
```

Reply:
done in [ca55348](https://github.com/NVIDIA/TensorRT-Model-Optimizer/pull/403/commits/ca5534883b9b4a8f596cf6a981c49f068927cc86)